src/rendering/RenderUtil.cc

```diff
@@ -1963,6 +1963,10 @@ void RenderUtilPrivate::RemoveRenderingEntities(
 /////////////////////////////////////////////////
 void RenderUtil::Init()
 {
+  // Already initialized
+  if (nullptr != this->dataPtr->scene)
```

Comment:
```suggestion
  if (this->dataPtr->scene)
```

Reply:
I don't think there's harm in being more explicit when checking for null pointers. Some classes may have custom comparison operators that don't return what we may expect. Do you think there's an advantage to leaving the `nullptr` out?